ui/app/AppLayouts/Profile/views/DevicesView.qml

```diff
@@ -17,6 +17,7 @@ import "../stores"
 SettingsContentBase {
     id: root
 
+    property var devicesList
```

Comment:
Not sure about the need of this property. Why not use directly the `store.deviceModel`?